src/doc/man/includes/options-locked.md

```diff
@@ -1,5 +1,5 @@
 {{#option "`--frozen`" "`--locked`"}}
-Either of these flags requires that the `Cargo.lock` file is
+Either of these flags require that the `Cargo.lock` file be
```

Comment:
`requires` should be singular, as it agrees with “Either,” which is a singular subject, no?